amd/src/package_search/mutations.js

```diff
@@ -85,8 +85,10 @@ export default class {
         state.general.loading = true;
         state.general.query = (typeof args.query === "string") ? args.query : state.general.query;
         state.general.tags = [];
-        state.general.sort = args.sort || state.general.sort;
-        state.general.order = args.order || state.general.order;
+        state.general.sorting = {
+            sort: args.sort || state.general.sorting.sort,
+            order: args.order || state.general.sorting.order,
+        };
```

Comment:
Ich bin mir gerade nicht sicher, ob diese Zuweisung die Watcher für sort und order auslösen würde. Ich meine mich zu erinnern, das man Object.assign verwenden muss, um Objekte im State zu ändern. Kann aber auch sein, dass ich mich irre. Hast du das ausprobiert, z.b. mit einem Watcher, der in die Konsole loggt?

Reply:
Ich habe das getestet und nur das Event `general.sorting:updated` wird ausgelöst. `general.sorting.sort` oder `general.sorting.order` dürften bei einer Modifizierung auch kein Event auslösen, da sie zu tief im Objekt liegen.

Reply:
Ok, ich persönlich fände das verwirrend. Das Ziel von der Zeile ist ja eigentlich, `sort` und `order` zu ändern. Aber für die beiden Felder wird kein watcher ausgelöst. Wenn das von dir so gewollt ist, ok.